test/remote.js

```diff
@@ -36,8 +36,7 @@ t.test('start server', t => {
       res.end(Buffer.alloc(2048))
     } else if (req.url === '/timeout') {
       res.statusCode = 200
-      res.write('just a second... ')
-      setTimeout(() => res.end('still there?'), 1000).unref()
+      // your call is important to us
```

Comment:
nit: not sure this is important?

Reply:
It is, on travis 1/6th of the time, apparently :)

Reply:
Perhaps a more verbose not for future folks reading through this because they got a failure?